packages/app-page-builder/src/pageEditor/config/blockEditing/BlockPreview.tsx

```diff
@@ -46,6 +45,9 @@ const BlockPreview: React.FC<BlockPreviewProps> = props => {
                 {onDelete && (
                     <Styled.DeleteBlock>
                         <ConfirmationDialog
+                            // We need to have this z-index because without it Delete Block Dialog will be rendered below.
```

Comment:
Please recheck this grammar. I think the dot after the `rendered below` is a mistake?

Sorry for the nitpick, but yeah, we're OSS, and companies tend to check our code sometimes before making their decision. I know this is just a drop in the sea, but yeah, gotta start somewhere. :)